web/src/components/contactForm.js

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import tw from 'twin.macro';
+
+const ContactWrapper = styled.div`
+    ${tw`text-center`}
+`;
+
+const StyledForm = styled.form`
+    ${tw`flex flex-col items-center`}
+`;
+
+const StyledHeading = styled.h1`
+    ${tw`font-bold mb-8 relative text-4xl text-black-3 leading-tight`}
+`;
+const InputWrapper = styled.div`
+    ${tw`flex gap-1`}
+`;
+
+const StyledButton = styled.button`
+    ${tw`cursor-pointer bg-green py-2 px-6 text-white max-w-max mt-16 text-xl tracking-widest`}
+
+    transition: background 500ms ease-out;
+
+    // &:hover {
+    //     background: #027874;
+    // }
+`;
+
+const StyledInput = styled.input`
+    ${tw`border-2`}
+`;
+
+const StyledLabel = styled.label`
+    ${tw`flex flex-col mb-4 mx-4 text-sm`}
+`;
+
+export default function ContactForm() {
+    const [formData, setFormData] = useState(null);
+    const [submitForm, setSubmitForm] = useState(false);
+
+    const encode = (data) => {
+        return Object.keys(data)
+            .map(
+                (key) =>
+                    `${encodeURIComponent(key)}=${encodeURIComponent(
+                        data[key],
+                    )}`,
+            )
+            .join('&');
+    };
+
+    const handleChange = (event) => {
+        setFormData({ ...formData, [event.target.name]: event.target.value });
+    };
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        const form = e.target;
+        fetch('/', {
+            method: 'POST',
+            headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+            body: encode({
+                'form-name': form.getAttribute('name'),
+                ...formData,
+            }),
+        })
+            .then(() => setSubmitForm(true))
+            .catch((error) => error);
+    };
+
+    return (
+        <ContactWrapper>
+            <StyledHeading>Request a Demo</StyledHeading>
+            {submitForm ? (
+                <StyledHeading>Thank you!</StyledHeading>
+            ) : (
+                <StyledForm
+                    method="post"
+                    onSubmit={handleSubmit}
+                    data-netlify="true"
+                    data-netlify-honeypot="bot-field"
```

Comment:
I think you need to add an input to correspond with this. And I'm not sure it needs to be prepended with `data-`. Check the [docs](https://docs.netlify.com/forms/spam-filters/#honeypot-field).

Reply:
@asasmith i did add the input, its below this. this is the form portion. I am not sure what you mean by this? I have double checked the docs a few times and I have everything it calls for

Reply:
I have checked the docs. I will remove data but had found that syntax in a netlify blog post so I am pretty sure it's good.. https://www.netlify.com/blog/2017/07/20/how-to-integrate-netlifys-form-handling-in-a-react-app/

Reply:
2017 though.